vitest.config.ts

```diff
@@ -24,6 +24,7 @@ export default defineConfig({
         'playground/**',
         'site/**',
         'create-onchain/**',
+        '**/**.test.tsx',
```

Comment:
Intentional?

Reply:
Yes. The code coverage was listing test files and counting them towards coverage %.  I had couple of my test files noted as covered 99.97% which prevented me from creating PR. (Its likely due to helper objects, functions that you create in a test files).